server/db/models/Gig.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function (sequelize, DataTypes) {
   var Gig = sequelize.define('Gig', {
     title: DataTypes.STRING,
-    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
+    type: DataTypes.STRING,
```

Comment:
Did you change the enum data type to a string to add flexibility to our event types? I think enum would be a good choice if we have 5 or less event types. happy to discuss